consumer_group.go

```diff
@@ -433,7 +433,24 @@ func (c *consumerGroup) joinGroupRequest(coordinator *Broker, topics []string) (
 		req.Version = 1
 		req.RebalanceTimeout = int32(c.config.Consumer.Group.Rebalance.Timeout / time.Millisecond)
 	}
-	if c.groupInstanceId != nil {
+	if c.config.Version.IsAtLeast(V0_11_0_0) {
+		req.Version = 2
+	}
+	if c.config.Version.IsAtLeast(V0_11_0_0) {
+		req.Version = 2
+	}
+	if c.config.Version.IsAtLeast(V2_0_0_0) {
+		req.Version = 3
+	}
+	// XXX: protocol states "Starting from version 4, the client needs to issue a
```

Comment:
the `if` below this comment sets `req.Version = 5` so this must work or already be broken?

Reply:
Yes, although I’ve not personally tested the V5 and static membership stuff so I didn’t want to introduce any breakage until I’d tested it